.github/workflows/docbuild-and-upload.yml

```diff
@@ -89,3 +89,10 @@ jobs:
         name: website
         path: web/build
         retention-days: 14
+
+    - name: Trigger web/doc preview
+      run: curl -X POST https://pandas.pydata.org/preview/submit/$RUN_ID/$PR_ID/
```

Comment:
Does this endpoint have any type of rate limiting or session management? Not sure how much we care about a bad actor trying to take it down

Reply:
Very good point. We're using Cloudflare, hopefully that can help us prevent some problems. The service itself is implemented in asynchronous rust, and it's very efficient, should be able to support quite a big work load. That being said, your comment makes me think of some things that can be improved. Happy to discuss them in private. In any case, I think it's good as it is to start, and if there are attacks, we can consider options, including pausing the service until we can prevent the problem.

Reply:
Sounds good. As long as you feel like there are other ways to mitigate and react happy to defer to your expertise. Change lgtm overall